src/test/java/nodejs/flows/albums/buyAlbum.feature

```diff
@@ -25,4 +25,4 @@ Feature: buying albums
     And request ''
```

Comment:
I suppose you are sending the request with ' ' to force the API to give you 422, correct?